build-tools/packages/build-tools/src/repoPolicyCheck/handlers/index.ts

```diff
@@ -8,6 +8,7 @@ import { handlers as copyrightFileHeaderHandlers } from "./copyrightFileHeader";
 import { handler as dockerfilePackageHandler } from "./dockerfilePackages";
 import { handler as fluidCaseHandler } from "./fluidCase";
 import { handlers as lockfilesHandlers } from "./lockfiles";
+import { handler as noJsFileHandler } from "./noJsFiles";
```

Comment:
We should always be using full paths now (okay if some aren't in my opinion until package is wholly updated; just good practice) Okay not to change.

Reply:
Prefer to fix these sorts of things all at once so someone else doesn't see it half one way and half another and get confused as to why they're inconsistent.